packages/vite/package.json

```diff
@@ -69,17 +69,17 @@
     "@types/etag": "^1.8.1",
     "@types/less": "^3.0.3",
     "@types/mime": "^2.0.3",
-    "@types/node": "^15.14.9",
+    "@types/node": "^16.11.6",
     "@types/resolve": "^1.20.1",
-    "@types/sass": "~1.16.1",
+    "@types/sass": "~1.43.0",
     "@types/stylus": "^0.48.36",
     "@types/ws": "^7.4.7",
     "@vue/compiler-dom": "^3.2.21",
     "acorn": "^8.5.0",
     "acorn-class-fields": "^1.0.0",
     "acorn-static-class-features": "^1.0.0",
     "builtin-modules": "^3.2.0",
-    "cac": "6.7.3",
+    "cac": "6.7.9",
```

Comment:
There is a breaking change in 6.7.10+ for us :thinking:

Reply:
dependency should be added to renovate dependency ignore list